Enable debug options and modify the use of CLFAGS (carried over
from the previous version's patch).

Upstream-Status: Pending

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

diff -urN insserv-1.14.0.orig//Makefile insserv-1.14.0//Makefile
--- insserv-1.14.0.orig//Makefile	2010-04-29 03:33:40.000000000 -0700
+++ insserv-1.14.0//Makefile	2010-11-28 17:09:20.537996190 -0800
@@ -8,27 +8,16 @@
 INITDIR  =	/etc/init.d
 INSCONF  =	/etc/insserv.conf
 #DESTDIR =	/tmp/root
-#DEBUG	 =	-DDEBUG=1 -Wpacked
+DEBUG	 =	-DDEBUG=1 -Wpacked
 DEBUG	 =
-ISSUSE	 =	-DSUSE
+#ISSUSE	 =	-DSUSE
 DESTDIR	 =
 VERSION	 =	1.14.0
 DATE	 =	$(shell date +'%d%b%y' | tr '[:lower:]' '[:upper:]')
 
-#
-# Architecture
-#
-ifdef RPM_OPT_FLAGS
-	  COPTS = -g $(RPM_OPT_FLAGS)
-else
-	   ARCH = $(shell uname -i)
-ifeq ($(ARCH),i386)
-	  COPTS = -g -O3 -mcpu=i586 -mtune=i686
-else
-	  COPTS = -g -O2
-endif
-endif
-	 CFLAGS = -W -Wall $(COPTS) $(DEBUG) $(LOOPS) -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
+COPTS = -O2 -g
+
+	 MY_CFLAGS = $(CFLAGS) -W -Wall $(COPTS) $(DEBUG) $(LOOPS) -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 		  $(ISSUSE) -DINITDIR=\"$(INITDIR)\" -DINSCONF=\"$(INSCONF)\" -pipe
 	  CLOOP = -falign-loops=0
 	LDFLAGS = -Wl,-O,3,--relax
@@ -75,13 +64,13 @@
 all:		$(TODO)
 
 insserv:	insserv.o listing.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
+	$(CC) $(MY_CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 listing.o:	listing.c listing.h config.h .system
-	$(CC) $(CFLAGS) $(CLOOP) -c $<
+	$(CC) $(MY_CFLAGS) $(CLOOP) -c $<
 
 insserv.o:	insserv.c listing.h config.h .system
-	$(CC) $(CFLAGS) $(CLOOP) -c $<
+	$(CC) $(MY_CFLAGS) $(CLOOP) -c $<
 
 listing.h:	.system
 
@@ -119,10 +108,10 @@
 -include	.depend.listing .depend.insserv
 
 .depend.listing::	listing.c listing.h
-	@$(CC) $(CFLAGS) -M listing.c >$@ 2>/dev/null
+	@$(CC) $(MY_CFLAGS) -M listing.c >$@ 2>/dev/null
 
 .depend.insserv::	insserv.c listing.h
-	@$(CC) $(CFLAGS) -M insserv.c >$@ 2>/dev/null
+	@$(CC) $(MY_CFLAGS) -M insserv.c >$@ 2>/dev/null
 
 endif
 
